src/pages/documentation/mobile-core/event-reference.md

```diff
@@ -1,26 +0,0 @@
-# Mobile Core event reference
-
-## Events dispatched
-
-The following event is dispatched by the Mobile Core extension:
-
-### Generic data OS event
-
-This event is generated by the Mobile Core extension when the `collectLaunchInfo` API is called.
-
-#### Event details
-
-| Event Type | Event Source | Paired | Direction |
-| :--- | :--- | :--- | :--- |
-| com.adobe.eventType.generic.data | com.adobe.eventSource.os | No | N/A |
```

Comment:
@prudrabhat are these materials already migrated to the core repo(s)?

Reply:
These haven't yet been moved to the core github repo. Based on what I remember from the core squad discussion, we plan to add them to the repo once the doc tasks are done.

Reply:
👍 that should be fine, just a reminder that when you do that you might need to refer to some of these PRs since this content will no longer be available in this repo